modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -178,29 +177,7 @@ private Connection wrapConnection(final Connection connection, final JdbcDatabas
     private void runInitScriptIfRequired(final ConnectionUrl connectionUrl, DatabaseDelegate databaseDelegate) throws SQLException {
         if (connectionUrl.getInitScriptPath().isPresent()) {
             String initScriptPath = connectionUrl.getInitScriptPath().get();
-            try {
-                URL resource;
-                if (initScriptPath.startsWith(FILE_PATH_PREFIX)) {
```

Comment:
note: this part was special for repeated code - moved inside of `ScriptUtils.runInitScript`